olympus-oir/frame_manager.py

```diff
@@ -52,8 +52,15 @@ def get_pixel_value_tm(self, myDataCnt):
         return self.m_pucImageBuffer_asWORD[myDataCnt]
 
     def pucBuffer_to_WORD_TM(self):
-        self.m_pucImageBuffer_asWORD = cast(self.m_pucImageBuffer, POINTER(c_uint16))
+        # TODO: The width and height should be obtained externally as appropriate.
+        #   (If it is difficult to obtain them here, they should be obtained elsewhere.)
+        buffer_size = c_uint16 * 512 * 512
```

Comment:
Please respond to this TODO.

Reply:
@itutu-tienday 
Do you mean the width and height of the image from AreaImageSize?

Reply:
@tuanna26 
I would assume that the information obtained with GetLevelImageSize would be used, but if the appropriate information is also obtained with AreaImageSize, then I think it is OK.